app/views/handlers/list_collector_content.py

```diff
@@ -0,0 +1,17 @@
+from typing import Any
+
+from app.views.handlers.list_collector import ListCollector
+from app.views.handlers.question import Question
+
+
+class ListCollectorContent(ListCollector):
+    def _get_additional_view_context(self) -> Any:
```

Comment:
missing proper return type

Reply:
This might return any as content key might not always be there in rendered block but we got around it by accessing using .get() method.